<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyMarkDownHelper</class>
 <widget class="QMainWindow" name="PyMarkDownHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>帮助</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget_lift">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>浏览</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_search">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Search">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/images/images/find.png</normaloff>:/images/images/find.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeView" name="treeView_Helper">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>标签</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget_right">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_Browser">
        <attribute name="title">
         <string>浏览</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenDirectory"/>
  </widget>
  <action name="actionOpenDirectory">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/bokeh.png</normaloff>:/images/images/bokeh.png</iconset>
   </property>
   <property name="text">
    <string>打开目录(&amp;o)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
